lib/tasks_private/spec_helper.rake

```diff
@@ -142,18 +142,10 @@ class PopulateTower
     data = {"name" => "hello_gce_cred", "kind" => "gce", "username" => "hello_gce@gce.com", "ssh_key_data" => ssh_key_data, "project" => "squeamish-ossifrage-123", "organization" => organization['id']}
     _gce_credential = create_obj(uri, data)
 
-    # create cloud rackspace cred
-    data = {"name" => "hello_rax_cred", "kind" => "rax", "username" => "admin", "password" => "abc", "organization" => organization['id']}
-    _rax_credential = create_obj(uri, data)
-
     # create cloud azure(RM) cred
     data = {"name" => "hello_azure_cred", "kind" => "azure_rm", "username" => "admin", "password" => "abc", "subscription"  => "sub_id", "tenant" => "ten_id", "secret" => "my_secret", "client" => "cli_id", "organization" => organization['id']}
     _azure_credential = create_obj(uri, data)
 
-    # create cloud azure(Classic) cred
-    data = {"name" => "hello_azure_classic_cred", "kind" => "azure", "username" => "admin", "ssh_key_data" => ssh_key_data, "organization" => organization['id']}
-    _azure_credential = create_obj(uri, data)
-
     # create cloud satellite6 cred
     data = {"name" => "hello_sat_cred", "kind" => "satellite6", "username" => "admin", "password" => "abc", "host"  => "s1.sat.com", "organization" => organization['id']}
     _azure_credential = create_obj(uri, data)
```

Comment:
A favor to ask for a knit that's not related to your changes: can you rename this `_azure_credential` to `_sat6_credential`?

Reply:
Sure. Updated that, I will include it with rebase.